ansible/files/whisks_design_document_for_activations_db.json

```diff
@@ -2,7 +2,7 @@
   "_id": "_design/whisks",
   "views": {
     "all": {
-      "map": "function (doc) {\n  var PATHSEP = \"/\";\n  var isActivation = function (doc) { return (doc.activationId !== undefined) };\n  var collection = function (doc) {\n    if (isActivation(doc)) return \"activations\";\n    return \"??\";\n  };\n\n  if (isActivation(doc)) try {\n    var ns = doc.namespace.split(PATHSEP);\n    var root = ns[0]; ns.shift();\n    var type = collection(doc);\n    var date = new Date(doc.start || doc.updated);\n    var value = {collection: type, namespace: doc.namespace, name: doc.name, version: doc.version, publish: doc.publish, annotations: doc.annotations};\n    value.activationId = doc.activationId;\n    emit([root, date], value);\n  } catch (e) {}\n}\n"
+      "map": "function (doc) {\n  var PATHSEP = \"/\";\n  var isActivation = function (doc) { return (doc.activationId !== undefined) };\n  var collection = function (doc) {\n    if (isActivation(doc)) return \"activations\";\n    return \"??\";\n  };\n\n  if (isActivation(doc)) try {\n    var ns = doc.namespace.split(PATHSEP);\n    var root = ns[0]; ns.shift();\n    var type = collection(doc);\n    var date = new Date(doc.start || doc.updated);\n    var value = {collection: type, namespace: doc.namespace, name: doc.name, version: doc.version, publish: doc.publish, annotations: doc.annotations, entityType: doc.entityType};\n    value.activationId = doc.activationId;\n    emit([root, date], value);\n  } catch (e) {}\n}\n"
```

Comment:
let's avoid any changes to the activation view unless there's a very good reason for it.